examples/dmd/local_tw_csv.cpp

```diff
@@ -15,13 +15,15 @@
 // =================================================================================
 //
 // Local serial DMD command:
-//   mpirun -np 8 local_tw_csv -o local_csv_serial -rdim 16 -dtc 0.01
+//   mpirun -np 8 local_tw_csv -o local_csv_serial -rdim 16 -dtc 0.01 -offline
```

Comment:
offline/online is needed in examples/dmd/parametric_tw_csv.cpp but not this file.

Reply:
Absolutely. I remove the redundant options.